WebHostLib/tracker.py

```diff
@@ -11,9 +11,38 @@
 from WebHostLib import app, cache, Room
 from Utils import restricted_loads
 from worlds import lookup_any_item_id_to_name, lookup_any_location_id_to_name
+from worlds.AutoWorld import AutoWorldRegister
 from MultiServer import get_item_name_from_id, Context
 from NetUtils import SlotType
 
+
+class PlayerTracker:
+
+    template: str = 'playerTracker.html'
+    icons: Dict[str, str] = {}
+    regions: Dict[str, List[str]] = {}
+    checks_done: Dict[str, Set[str]] = {}
```

Comment:
Regarding @Ijwu 's comment/change request: not defining the instance variables anywhere does not provide typing information. Pycharm *may* be smart detecting what is assigned in `__init__` (not sure), but there is actually no typing information being set.
[PEP 526](https://peps.python.org/pep-0526/#class-and-instance-variable-annotations) says that the removed block would in fact define instance variables, not class variables. And that's also what is being used in BaseClasses.py's classes. So the removed block should be re-added.

In theory you could do `self.room: Room = room` in `__init__`, but currently (py3.10) this seem to only do type checking, and does not store any typing information in the class or instance afaict.

Reply:
Thank you for informing us (me, especially) on this. I am working off outdated knowledge of Python and am glad to update it as time passes. I genuinely appreciate the response. @alwaysintreble I'm sorry for flip-flopping on you, are you okay with re-implementing this? It's not my intent to complicate the review process, but sometimes people are wrong. Especially if it's me, I'm wrong a lot. 😅